security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -69,6 +70,15 @@ const (
 	// Timeout the K8s update/delete notification threads. This is to make sure to unblock the
 	// secret watch main thread in case those child threads got stuck due to any reason.
 	notifyK8sSecretTimeout = 30 * time.Second
+
+	// The well-known path for an existing certificate chain file
+	existingCertChainFile = "/etc/certs/cert-chain.pem"
```

Comment:
Please use "./etc/..." - with istiod and agent we now support running and debugging on local machine as non-root user.

In docker we run with CWD=/, so it's same thing.

Reply:
Done.